src/Umbraco.Web.UI.Client/src/common/directives/components/forms/umbcheckbox.directive.js

```diff
@@ -61,6 +62,7 @@
             value: "@",
             name: "@",
             text: "@",
+            labelKey: "@",
```

Comment:
`umbButton` has `label` and `labelKey` properties.
https://github.com/umbraco/Umbraco-CMS/blob/v8/dev/src/Umbraco.Web.UI.Client/src/common/directives/components/buttons/umbbutton.directive.js#L94-L95

Would it be better naming this `textKey`? to match the property `text` since we can't change the naming of the `text` property at least not before v8.3